code/modules/uplink/uplink_items.dm

```diff
@@ -1717,6 +1717,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/box/hug/reverse_revolver
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/role_restricted/cluwne
```

Comment:
I think this PR is fine, but I don't want these as a syndicate item.

Given the mutated/weird nature of these, I think they should remain for admins to use in portal storms and special events.

I would also allow a clown science station space ruin with a few of the grenades etc

but it's my view that this makes no sense as a syndicate item